Geospatial/Travel Times/travel times map.R

```diff
@@ -0,0 +1,71 @@
+# needed for managing travel times data
+library(tidyverse)
+library(sf)
+library(osrm)
+library(sfheaders)
+
+# Needed for reading in data
+library(arrow)
+
+# Needed for producing map
+library(leaflet)
+library(glue)
+
+# Load in travel times functions - change the filepath to wherever you have saved the file travel times functions.R
+source("Geospatial/Travel Times/travel times functions.R")
+
+# Pick a Locality - this can be changed to whichever locality is needed
+locality <- "Kilmarnock"
+
+# Read in the locality shapefile
+shapefiles_folder <- file.path("/conf/linkage/output/lookups/Unicode/Geography/Shapefiles")
```

Comment:
```suggestion
lookups_folder <- file.path("/conf/linkage/output/lookups/Unicode")
shapefiles_folder <- file.path(lookups_folder, "Geography/Shapefiles")
```
`lookups_folder` can now also be used in a couple of other places below.